bin/reth/src/debug_cmd/build_block.rs

```diff
@@ -245,12 +245,20 @@ impl Command {
             suggested_fee_recipient: self.suggested_fee_recipient,
             // TODO: add support for withdrawals
             withdrawals: None,
+            #[cfg(feature = "optimism")]
+            optimism_payload_attributes: reth_rpc_types::engine::OptimismPayloadAttributes {
+                transactions: None,
+                no_tx_pool: None,
+                gas_limit: None,
+            },
```

Comment:
```suggestion
            optimism_payload_attributes: reth_rpc_types::engine::OptimismPayloadAttributes::default(),
```

Reply:
Oops good call, added a default derivation.

Addressed in https://github.com/anton-rs/op-reth/pull/145